packages/aragon-client/package.json

```diff
@@ -58,7 +60,8 @@
     "babel": "inherit"
   },
   "dependencies": {
-    "rxjs": "^5.5.6",
-    "@aragon/messenger": "^1.0.0-beta.6"
+    "@aragon/messenger": "^1.0.0-beta.6",
+    "@babel/runtime": "^7.1.2",
```

Comment:
Not sure if `aragon-client` and `aragon-messenger` need the `@babel/runtime` library, but their old babel configs were strangely using the `transform-runtime` plugin without including this dependency.

Reply:
Yes I saw the same in `aragon/aragon`, it was probably a dependency of another module.